content/posts/2021/01/2021-06-14-outreachy-application-process/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: My Outreachy Application Process
+author: Cess wairimu
+date: 2021-06-14
+hero: ./images/outreachy.png
+excerpt: This blog outlines my Outreachy application process and some personal advise about the same.
+---
+
+# My Outreachy Application Process
+
+Outreachy provides a 3 month paid internship to work in Free and Open Source Software. It happens twice each year(in Summer and Winter) and targets people underrepresented in tech.I participated in the Outreachy Internship program during the 2018 Winter round.
+
+I first heard of Outreachy from a friend who had previously participated in the program. The reason why I decided to apply to  the program is because  I love open source and would like to work in open source in future, so Outreachy was a great place to get started and the experience would be good for my career generally. I signed up and went through the information on [the website](https://www.outreachy.org/) about the internship months before the applications were opened. I also signed up for a newsletter which informs you when the applications open. I went through past projects and organizations which gave me an idea of some the organizations which I would like to apply to. There are blogs from past interns talking about their application processes and they also offer tips on what worked based on their experiences. I read some of these which gave me an idea on what to expect. When the internship applications opened, I got an email telling me so :mailbox:
+
```

Comment:
@cesswairimu Would be nice if you add a sub-heading here eg
Application process steps
To get the reader's attention